hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/store/impl/ZookeeperFederationStateStore.java

```diff
@@ -1002,7 +1053,9 @@ public RouterRMTokenResponse removeStoredToken(RouterRMTokenRequest request)
   @Override
   public RouterRMTokenResponse getTokenByRouterStoreToken(RouterRMTokenRequest request)
       throws YarnException, IOException {
+    request.getRouterStoreToken().getTokenIdentifier();
     throw new NotImplementedException("Code is not implemented");
```

Comment:
Didn't we implement it?

Reply:
This part of the code has been completed, I will confirm my submission record again.